pkg/microservice/aslan/core/environment/service/environment.go

```diff
@@ -84,19 +85,21 @@ type EnvStatus struct {
 }
 
 type EnvResp struct {
-	ProjectName string `json:"projectName"`
-	Status      string `json:"status"`
-	Error       string `json:"error"`
-	Name        string `json:"name"`
-	UpdateBy    string `json:"updateBy"`
-	UpdateTime  int64  `json:"updateTime"`
-	IsPublic    bool   `json:"isPublic"`
-	IsExisted   bool   `json:"is_existed"`
-	ClusterName string `json:"clusterName"`
-	ClusterID   string `json:"cluster_id"`
-	Production  bool   `json:"production"`
-	Source      string `json:"source"`
-	RegistryID  string `json:"registry_id"`
+	ProjectName string   `json:"projectName"`
```

Comment:
Use `project_name` to keep style consistent?

Reply:
we use projectName